api-reference/10 UI Components/dxScheduler/1 Configuration/onAppointmentClick.md

```diff
@@ -67,3 +68,18 @@ This object describes a series of appointments that belong to two owners and rep
             recurrenceRule: "FREQ=DAILY"
         } */
     }
+
+Besides objects common with the **appointmentData** field, the **targetedAppointmentData** field has the **displayStartDate** and **displayEndDate** date objects. These objects contain time-zone adjusted dates if you specify the [timeZone](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#timeZone) property. If the latter property is not specifies, the date objects mentioned above equal the **startDate** and **endDate** objects.
```

Comment:
- Do not use `Besides` 
- `objects common with the **appointmentData** field` Objects common with? What does this mean?
- `is not specifies` Can you clarify?
- `equal the **startDate** and **endDate** object` Does this mean they are the same? I'm not sure how one object can 'equal' another